tests/spring-service/simple-service-rls/values.yaml

```diff
@@ -29,38 +21,8 @@ ingress:
   overwriteCustomHttpErrors: "502"
   basicAuthSecretParameterName: myservice/auth
 
-timeouts:
-  progressDeadlineSeconds: 120
-  terminationTimeBeforeKillSeconds: 60
-  terminationTimeBeforeSigTermSeconds: 45
-
 env:
   copyOwnExternalBaseURLIn: "SERVICE_MYSERVICE_BASE_URL"
 
   springDatasourceFromSecretWithRLS: "rds/myservice"
   springRabbitMQSecret: true
-
-  fromSecret:
-    SOME_SECRET:
```

Comment:
interpretiere ich das hier richtig, dass du das RLS beispiel eingedampft hast, damit der Blick aufs wesentliche eher frei wird?

Reply:
yes!

Reply:
Thanks, I did consider doing this with the RLS PR, but now with the defaults here it makes perfect sense :)